bookkeeper-common/src/main/java/org/apache/bookkeeper/common/util/OrderedScheduler.java

```diff
@@ -377,6 +377,6 @@ protected ListeningExecutorService delegate() {
             public void execute(Runnable command) {
                 super.execute(timedRunnable(command));
             }
-        };
+        }
```

Comment:
just wondering why our checkstyle rule doesn't catch this?

Reply:
I could not find any check which can catch this violation.

Reply:
I could not find any check which can catch this violation.